### ISSUE TITLE
Yo is this line right?

### ISSUE THREAD

**Issue by Maxpridy** [Maxpridy/algorithm_solve#1]
https://github.com/Maxpridy/algorithm_solve/blob/6c0ddaa7bde29f9dc79a2fc92c6f7eb6487b03fc/train.py#L98

I found this line when I check validation loss.

**Comment by donghyeops** [comment]
huk. train_loss->val_loss !!!

**Comment by donghyeops** [comment]
I fixed it.
Also, I found a code that trains the model with validation dataset.
It's terrible mistake.
Sorry, I fixed it all.
